packages/clerk-js/src/ui/utils/phoneUtils.ts

```diff
@@ -115,3 +116,31 @@ export function stringToFormattedPhoneString(str: string): string {
   const parsed = parsePhoneString(str);
   return `+${parsed.code} ${formatPhoneNumber(parsed.number, parsed.pattern, parsed.code)}`;
 }
+
+type CountryData = {
+  searchTerm: string;
```

Comment:
It looks like this `searchTerm` was only added because you wanted to use `countyOptions` from the `PhoneInput.tsx`. As I commented above, lets not leak any UI concerns inside these helpers. Instead, lets remove CountryData altogether and use the country map directly here